src/OmegaPoker.sol

```diff
@@ -65,6 +65,8 @@ contract OmegaPoker {
 
             address _pip = registry.pip(_ilks[i]);
 
+            if (_pip == address(0)) { continue; }
+
             // OSM's and LP oracles have src() function
             (bool ok,) = _pip.call(abi.encodeWithSelector(srcselector));
```

Comment:
Wait, so `ok` would be true here even if `_pip` is `address(0)`? Otherwise I don't see how the added check makes any difference in execution here.

Reply:
Correct.

Reply:
Hmmmm I guess since there's no code at `address(0)` there's nothing to cause a revert so the EVM just thinks all went well under the hood. Super counterintuitive.

Reply:
![image](https://user-images.githubusercontent.com/2636663/201777710-44633273-4276-4d97-9c73-b62f9e52ab1e.png)

The new test passes even if I comment out the added line. Something is off.

Reply:
Yeah, so the original test fails because it allows a `pip` set to `address(0)` into the array.

Without this line, the `testPoke()` function fails because the pip for `TELEPORT-FW-A` is set to `address(0)` in the registry. 

`poke()` actually still works in this case so we might need to go a step further in this new test, iterate all of the post-refresh ilks, and revert if we see an `address(0)` in the list.

![Screenshot from 2022-11-14 17-32-22](https://user-images.githubusercontent.com/2374718/201782532-7a1ac8dc-4b12-49c0-ac4d-444c8c942032.png)

Reply:
Confirmed that the latest change fixes this.